pkg/datamodel-tests/src/implementations/version-1.1/db.ts

```diff
@@ -92,4 +111,26 @@ export const newDatabase = (db: PouchDB.Database): Database => {
 	return new Database(db);
 };
 
+/**
+ * A function that handles replication, returns resolve when replication is complete and reject otherwise
+ * @param local - local pouchdb instance
+ * @param remote - remote pouchdb instance
+ */
+
+const replicate = (database: { remote: PouchDB.Database; local: PouchDB.Database }) =>
+	new Promise<void>((resolve, reject) => {
+		database.local.replicate
+			.to(database.remote)
+			.on('complete', function () {
+				// yay, we're done!
+				console.log('Replication complete');
+				resolve();
+			})
+			.on('error', function (err) {
+				// boo, something went wrong!
+				console.log('could not replicate to remote db', err);
+				reject();
```

Comment:
Will this also replicate from the remote? (when I add a document in the remote db, I would like to see the update locally)

Reply:
This is a bit of a redundancy (you can simply import from `utils/pouchdb`

Reply:
Is this replication live...does it keep replicating as either of the dbs changes?